doc/api.rst

```diff
@@ -100,6 +113,7 @@ Modules
     list_runs
     run_model_on_task
     run_flow_on_task
+    run_exists
```

Comment:
This isn't in the `__all__` variable of `runs/__init__.py`. Could you please add it theres?

Reply:
If I understand this right, I believe it's there?
https://github.com/openml/openml-python/blob/apidocs/openml/runs/__init__.py#L25

Reply:
Not sure how I missed that, sorry.